### DISCUSSION TITLE
Distribution release for OTA

### DISCUSSION THREAD

**Discussion by kylepinionseb** [crowdin/github-action#199] [Q&A] [answered]
I've used this repo for Crowding Upload and Download translations in GitHubAction, and it is working well. 
Is there a way to go about calling the `distribution` command with it?
I'd like to schedule the OTA release to occur on a scheduled basis so the Cloud is updated regularly without someone having to hit the "Release" button on the crowding dashboard.
The documentation for it is https://developer.crowdin.com/api/v2/#operation/api.projects.distributions.release.post
and also https://crowdin.github.io/crowdin-cli/commands/crowdin-distribution
Maybe I have to do a custom GHA for this or alternate solution.

**Comment by TaniaMal** [comment]
Hi @kylepinionseb,

Let me check if there is such a possibility and get back to you

**Comment by andrii-bodnar** [comment, marked as answer]
Hi @kylepinionseb, you can run any Crowdin CLI command by using this Action, including `distribution`:

```yml
- name: crowdin action
  uses: crowdin/github-action@v1
  with:
    command: 'distribution release'
```

https://github.com/crowdin/github-action#crowdin-cli-command

**Reply by kylepinionseb** [reply]
Perfect, that will suffice. Thanks!
